widgets/package.json

```diff
@@ -1,10 +1,44 @@
 {
 	"name": "huggingface-widgets",
 	"version": "0.0.1",
+	"type": "module",
+	"main": "./package/InferenceWidget/InferenceWidget.svelte",
+	"exports": {
+		".": {
+			"import": "./package/InferenceWidget/InferenceWidget.svelte",
+			"require": "./package/InferenceWidget/InferenceWidget.svelte.js"
+		},
+		"./ModelPipelineIcon.svelte": "./package/ModelPipelineIcon/ModelPipelineIcon.svelte",
+		"./ModelPipelineIcon.svelte.js": "./package/ModelPipelineIcon/ModelPipelineIcon.svelte.js",
+		"./ModelPipelineTag.svelte": "./package/ModelPipelineTag/ModelPipelineTag.svelte",
+		"./ModelPipelineTag.svelte.js": "./package/ModelPipelineTag/ModelPipelineTag.svelte.js",
+		"./WidgetHeader.svelte": "./package/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte",
+		"./WidgetHeader.svelte.js": "./package/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte.js",
+		"./interfaces/DefaultWidget": "./package/interfaces/DefaultWidget.js",
+		"./interfaces/DefaultWidget.js": "./package/interfaces/DefaultWidget.js",
+		"./interfaces/Language": "./package/interfaces/Language.js",
+		"./interfaces/Language.js": "./package/interfaces/Language.js",
+		"./interfaces/Libraries": "./package/interfaces/Libraries.js",
+		"./interfaces/Libraries.js": "./package/interfaces/Libraries.js",
+		"./interfaces/Types": "./package/interfaces/Types.js",
+		"./interfaces/Types.js": "./package/interfaces/Types.js"
```

Comment:
I'm exporting `components` (InferenceWidget, ModelPipelineTag, ...) & `interfaces` (Types, Libraries, ...) in  `package.json` rather than creating `index.ts` files. @Pierrci please let me know what you think about this point